### DISCUSSION TITLE
How to add default values for forms in security schemes?

### DISCUSSION THREAD

**Discussion by nunomen** [fastapi/fastapi#8052] [Questions] [answered]
The API does not seem to support this, am I wrong?

![64032518-e87c7600-cb4a-11e9-8f23-fa6210f9c7dd](https://user-images.githubusercontent.com/9057382/64179060-c9d7f100-ce62-11e9-9531-8bfb499f2ccc.png)

**Comment by zamiramir** [comment, marked as answer]
see PR #499

**Comment by tiangolo** [comment]
Thanks for the help here @zamiramir , and thanks @littlebrat for coming back to close the issue :+1:
